lib/adequate_crypto_address/eth_tokens/aave.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module AdequateCryptoAddress
+  module EthTokens
+    class AAVE < Eth
```

Comment:
These class names need to be updated to camelcase. This gives the impression that it is a constant and not a class.

`Aave` and `AaveToken`.